install-deps-osx.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -e
+set -x
+brew update
+brew install gcc
+brew install sqlite
+brew install zlib
+brew install bzip2
+brew install cairo
+brew install dbus
+brew install expat
+brew install fontconfig
+brew install freetype
+brew install gettext
+brew install glib
+brew install gmp
+brew install hdf5
+brew install icu4c
+brew install jpeg
+brew install libarchive
+brew install curl
+brew install libedit
+brew install libffi
+brew install libpng
+brew install libsodium
+brew install libssh2
+brew install libtiff
+brew install libxml2
+brew install libxslt
+brew install libomp
+brew install lz4
+brew install lzo
+brew install ncurses
+brew install pcre
+brew install qt@5
+brew install snappy
+brew install tcl-tk
+brew install xz
+brew install libyaml
+brew install zeromq
+brew install zstd
+brew install python@3.10
+brew install openblas # replaces: pip install blas
```

Comment:
These are the equiv packages to the things in the original environment.yml that I couldn't get to build.  Some of them might not be used?